airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -789,49 +791,62 @@ def insert_all(self, project_id: str, dataset_id: str, table_id: str,
                 )
             self.log.error(error_msg)
 
-    def update_dataset(self, dataset_id: str,
-                       dataset_resource: Dict, project_id: Optional[str] = None) -> Dict:
-        """
-        Updates information in an existing dataset. The update method replaces the entire
-        dataset resource, whereas the patch method only replaces fields that are provided
-        in the submitted dataset resource.
-        More info:
-        https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets/update
+    @GoogleBaseHook.fallback_to_default_project_id
+    def update_dataset(
+        self,
+        fields: Sequence[str],
```

Comment:
This is a new parameter. There is no way to maintain backward compatibility. If yes, can you add a note in UPDATIND.md?

Reply:
Done